### PR TITLE
Implement tabs on Search page

### PR DESCRIPTION
**Pull request by szabolcs-keresztes**
- Search page with _Candidate details_ and _Advanced search_ tabs
- Candidate details search UI

IMG_0101

### CODE REVIEW

File: src/pages/test-results-search/components/tabs/tabs.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  ul {
+    list-style: none;
+    padding: 0;
+    margin-bottom: 15px;
+  }
+
+  li {
+    display: inline;
+    border: 1px solid map-get($colors-gds, 'gds-grey-4');
+    border-width: 1px 1px 1px 1px;
```
Comment (Cavalier07) [inline src/pages/test-results-search/components/tabs/tabs.scss]: can be shorthanded

File: src/pages/test-results-search/components/tabs/tabs.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    border-bottom-color: black;
```
Comment (Cavalier07) [inline src/pages/test-results-search/components/tabs/tabs.scss]: can this come from color mapping?

File: src/pages/test-results-search/components/tabs/tabs.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    padding: 16px;
+    background-color: map-get($colors-gds, 'gds-grey-4');
+  }
+
+  li.active {
+    background-color: map-get($colors, 'mes-white');
+    border: solid;
+    border-width: 1px 1px 1px 1px;
```
Comment (Cavalier07) [inline src/pages/test-results-search/components/tabs/tabs.scss]: Can be shorthanded

File: src/pages/test-results-search/components/tabs/tabs.scss (hunk 1 above)
Comment (szabolcs-keresztes) [reply src/pages/test-results-search/components/tabs/tabs.scss]: done

File: src/pages/test-results-search/components/tabs/tabs.scss (hunk 3 above)
Comment (szabolcs-keresztes) [reply src/pages/test-results-search/components/tabs/tabs.scss]: done

File: src/pages/test-results-search/components/tabs/tabs.scss (hunk 2 above)
Comment (szabolcs-keresztes) [reply src/pages/test-results-search/components/tabs/tabs.scss]: done

**Pull request merged**